### PR TITLE
feat: add user rich rank list page

### LINKED ISSUE #132: The ranking pages do not display correctly
On the main  page, the ranking pages, for example ranking of wealth, are not operating. After clicking, the page show nothing.
Please fix this bug.
And in the posts, if you upload a video links, you cannot size up the video in PC. But it can be done in mobile phones.

### PR DESCRIPTION
**Pull request by oceanlvr**
fix: #132 

add API:

* getMemberRichList, get the top of 15 rich users.

image

Signed-off-by: oceanlvr <657531018@qq.com>

### CODE REVIEW

**Comment by oceanlvr** [comment]
@hsluoyz please review:) thanks!

File: routers/router.go [hunk 1]
```diff
@@ -80,6 +80,7 @@ func initAPI() {
 	beego.Router("/api/update-member-editorType", &controllers.APIController{}, "POST:UpdateMemberEditorType")
 	beego.Router("/api/get-member-editorType", &controllers.APIController{}, "GET:GetMemberEditorType")
 	beego.Router("/api/update-member-email-reminder", &controllers.APIController{}, "POST:UpdateMemberEmailReminder")
+	beego.Router("/api/get-members-rankRich", &controllers.APIController{}, "GET:GetMembersRankRich")
```
Comment (hsluoyz) [inline routers/router.go]: /api/get-ranking-rich

File: web/src/App.js [hunk 2]
```diff
@@ -54,6 +54,7 @@ import MoveTopicNodeBox from "./main/MoveTopicNodeBox";
 import EditBox from "./main/EditBox";
 import ForgotBox from "./main/ForgotBox";
 import FilesBox from "./main/FilesBox";
+import RichBox from "./main/RichBox";
```
Comment (hsluoyz) [inline web/src/App.js]: RankingRichBox

File: web/src/locales/en.json [hunk 3]
```diff
@@ -100,6 +100,7 @@
     "File library": "File library"
   },
   "balance": {
+    "Rich rank": "rich rank",
```
Comment (hsluoyz) [inline web/src/locales/en.json]: "Rich ranking": "Rich ranking",

File: object/member.go [hunk 4]
```diff
@@ -65,6 +67,15 @@ func GetMembers() []*Member {
 	return members
 }
 
+func GetMembersRankRich() []*Member {
```
Comment (hsluoyz) [inline object/member.go]: GetRankingRich

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/backend/MemberBackend.js [hunk 5]
```diff
@@ -172,3 +172,10 @@ export function getMemberEditorType() {
     credentials: "include",
   }).then((res) => res.json());
 }
+
+export function getMemberRichList() {
```
Comment (hsluoyz) [inline web/src/backend/MemberBackend.js]: getRankingRichList

File: web/src/main/RankingRichBox.js [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+          src={Setting.getStatic("/static/img/bronze@2x.png")}
+          height="16"
+          alt="B"
+          border="0"
+        />
+      </div>
+    );
+  }
+
+  renderRichList() {
```
Comment (hsluoyz) [inline web/src/main/RankingRichBox.js]: render()

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**